types/src/dkg/mod.rs

```diff
@@ -146,10 +154,22 @@ impl OnChainConfig for DKGState {
     const TYPE_IDENTIFIER: &'static str = "DKGState";
 }
 
+#[derive(Clone, Debug, Default)]
+pub struct RoundingSummary {
+    pub method: String,
+    pub output: DKGRoundingProfile,
```

Comment:
how does the debug print for this look like?

Reply:
it's a bunch of statistics, thresholds, and the final weight vec.